packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 48.10.0 - April 9, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Added a tree view visualization to the [conflict detection beta](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/detect-conflicts) ([PR #2096](https://github.com/forcedotcom/salesforcedx-vscode/pull/2096))
```

Comment:
@brpowell , can we say tree visualization instead of tree view visualization?
Added tree visualization to conflict detection beta.